hooks/hookexecution/executor.go

```diff
@@ -50,3 +51,26 @@ func (executor *HookExecutor) ExecuteEntrypointStage(req *http.Request, body []b
 
 	return payload.Body, reject
 }
+
+func (executor *HookExecutor) ExecuteRawAuctionStage(requestBody []byte, account *config.Account) ([]byte, *RejectError) {
+	handler := func(
+		ctx context.Context,
+		moduleCtx *hookstage.ModuleContext,
+		hook hookstage.RawAuction,
+		payload hookstage.RawAuctionPayload,
+	) (hookstage.HookResult[hookstage.RawAuctionPayload], error) {
+		return hook.HandleRawAuctionHook(ctx, moduleCtx, payload)
+	}
+
+	executor.InvocationCtx.Stage = hooks.StageRawAuction
+	executor.InvocationCtx.Account = account
+	executor.InvocationCtx.AccountId = account.ID
```

Comment:
I propose to set these two fields inside `parseRequest` func in `auction.go` file, right after we got account instead of this place.
For example, if function `ExecuteRawAuctionStage` will be removed, the later hooks won't be able to access account anymore. But if we set account outside of this func, account will be still accessible for other hooks.